ScreenManager.lua

```diff
@@ -124,6 +135,9 @@ end
 function ScreenManager.init( nscreens, screen, ... )
     stack = {};
     screens = nscreens;
+
+    isValidScreen( screen );
```

Comment:
I put this here just so that the error points to the line where you called `ScreenManager:init` and not to:
```line 139 of ScreenManager.lua```